tests/test_keras.py

```diff
@@ -179,3 +179,37 @@ def test_keras_None_model_file_skip_load(
     )
 
     assert load_weights.call_count == 0
+
+
+def test_patch_tensorboard_keras_callback(tmp_dir, xor_model, mocker):
+    import gorilla
+    import tensorflow as tf
+
+    from dvclive.tensorboard import patch_tensorboard
+
+    scalar = mocker.spy(tf.summary, "scalar")
+    image = mocker.spy(tf.summary, "image")
+
+    patches = patch_tensorboard(path="logs")
+
+    model, x, y = xor_model()
+
+    model.fit(
+        x,
+        y,
+        epochs=2,
+        batch_size=1,
+        callbacks=[tf.keras.callbacks.TensorBoard()],
+    )
+
+    assert not scalar.call_count
+    assert not image.call_count
+
+    assert os.path.exists("logs")
+    logs, _ = read_logs("logs/scalars")
+
+    assert "epoch_accuracy" in logs
+    assert len(logs["epoch_accuracy"]) == 2
+
+    for patch in patches:
```

Comment:
So in case of using `dvclive` with `tensorboard` we would need to revert the patches after the execution in order to not use modified `tensorboard` later in our project? 

Shouldn't `patch_tensorboard` be a context manager that would revert the patch after exiting the context?

Reply:
I need to test it but I think that the patch doesn't persist across python executions.

Reply:
What about the same process? I think safest way would be to let the user control when the `dvclive` is patching the tensorboard.